english/headache/headache.md

```diff
@@ -0,0 +1,6 @@
+A throbbing pain pulses from the middle of your forehead. Squinting through the
+pain you reach to rub it and gain some relief, feeling the familiar texture of
+a band-aid as well as a strangely shaped lump. Not remembering how either got
+there you go to your bedroom mirror to inspect it further.
+
+Leaning into the mirror it seems to be a [spiral-shaped lump.](../spiral)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.